app/images/arrow-right-blue.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
SVG can be minified using https://jakearchibald.github.io/svgomg/

Reply:
Done.